apps/default/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10-bullseye
+RUN apt-get update -y && mkdir /app
+RUN apt-get install build-essential cmake python3-dev -y
+COPY . /app
+WORKDIR /app
+RUN pip3 install poetry
+RUN poetry config virtualenvs.create false
+RUN poetry install --only main
+ENTRYPOINT [ "streamsync", "run" ]
```

Comment:
Can you change dockerfiles to be writer run, please?

Reply:
removed dockerfiles

Reply:
Even better